app/static/src/scss/includes/_record.scss

```diff
@@ -198,3 +203,150 @@
         font-weight: 100;
     }
 }
+
+/* Default styles for desktop */
+
+@mixin on-mobile {
+    @media screen and (width <=810px) {
+      @content;
+    }
+
+    .record-details {
+
+            padding-top: 0;
+            display: flex;
+            flex-direction: column;
+            margin-top:0;
+    }
+  }
+
+.govuk-width-container--record {
+    margin-top: -2rem;
+}
+
+
+// DISABLE MOBILE STYLES //
+
+
+.govuk-grid-row {
+    &--mobile {
+        display: none;
+    }
+}
+
+
+
+
+
+//  MOBILE STYLES //
+
+@media screen and (width <=810px) {
+.govuk-grid-column-full__page-nav {
+    order: 2;
+    margin-bottom: 0.5rem;
+}
+
+
+.govuk-grid-row {
+    &--mobile {
+        display: flex;
+        flex-direction: column;
+    }
+}
+
+.govuk-grid-column-two-thirds {
+    &--record-table {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        order: 4;
+    }
+}
+
+
+.govuk-summary-list {
+    &--record-mobile {
+        display: auto;
+    }
+
+    &__key--mobile-table {
+        white-space: nowrap;
+        width:0;
+        text-align:right;
+        font-size: 1rem;
+    }
+
+    &__value--mobile {
+        font-size: 1rem;
+    }
+
+    &__row--mobile {
+        &:last-child {
+            border-bottom: 2px solid #000;
+        }
+        display: block;
+    }
+}
+
+    .govuk-width-container--record {
+      display: none;
+    }
+
+
+    .record-details {
+
+            padding-top: 0;
+            display: flex;
+            flex-direction: column;
+            margin-top:0;
+            order: 5;
+        }
+
+.rights-container {
+    display: flex;
+    order: 3;
+    margin-top: 1rem;
+    margin-bottom: 2rem;
+}
+
+.record-container {
+    display: flex;
+    flex-direction: column;
+    order: 5;
+    white-space: nowrap;
+}
+
+
+.mobile__search__container-block {
+    margin-bottom: -2rem;
+}
+
+
+.mobile__search__container {
+    display: flex;
+    flex-direction: row;
+    margin-top: -2rem;
+    align-items: center;
+    justify-content: center;
+    background-color: #f3f2f1;
+    width: 100%;
+    padding: 0;
+    position: absolute;
+    left: 0;
+    right: 0;
+    order: 1;
+}
+
+
+
+.record-mobile__search__container__content {
+    background-color: #f3f2f1;
```

Comment:
if this colour is used elsewhere let's make it a variable either in this file or another scss file that can be included

Reply:
I actually suggested this to george last week about colour variables

Reply:
I think we need to agree on approach to use variables, and then we can amend at all places

Reply:
yeah, as discussed offline, Pavan's created a ticket for renaming scss variables a whole